jinahub/encoders/audio/AudioCLIPEncoder/executor/audio_clip_encoder.py

```diff
@@ -90,14 +91,14 @@ def encode(
 
     def _create_embeddings(self, filtered_docs: Iterable):
         """Update the documents with the embeddings generated by AudioCLIP"""
-
         for d in filtered_docs:
             d.blob, d.tags['sample_rate'] = self._resample(
                 d.blob, dict(d.tags).get('sample_rate', None)
             )
-            audio = torch.Tensor(d.blob).unsqueeze(0)
-            embedding = self.model.encode_audio(audio=audio)[0]
-            d.embedding = embedding.cpu().numpy()
+
+        inputs  = torch.Tensor(filtered_docs.blobs).unsqueeze(1)
+        ((embeddings, _, _), _), _ = self.model(audio=inputs)
```

Comment:
Have we check out the functionality of `logits`? In the original demo, the author is using these `logits` for comparing similarities. https://github.com/AndreyGuzhov/AudioCLIP/blob/master/demo/AudioCLIP.ipynb

https://github.com/jina-ai/executors/blob/1825246962f557b2a2b8b8d513922c5c25fff007/jinahub/encoders/audio/AudioCLIPEncoder/executor/audio_clip/model/audioclip.py#L150-L175